msg/Detection2D.msg

```diff
@@ -15,3 +15,9 @@ BoundingBox2D bbox
 # The 2D data that generated these results (i.e. region proposal cropped out of
 #   the image). Not required for all use cases, so it may be empty.
 sensor_msgs/Image source_img
+
+# If this message was tracking result, this field set true.
+bool is_tracking
+
+# Unique ID which was set by tracker.
+uuid_msgs/UniqueID tracking_id
```

Comment:
Can we change the comments to the following, for better readability? (on both message definitions)
```
# If true, this message contains object tracking information.

# ID used for consistency across multiple detection messages.
```

Reply:
Please rename tracking_id to object_id because it can also be used in other contexts, where the result is not generated from a tracker. 
Moreover, I would remove the is_tracking field. This can be easily derived from the context, e.g. the publishing node or that tracking_id/object_id is different from 0 or "".

Reply:
@mintar I disagree with you. In my opinion, detector cannot set object ID because they did not do time series processing.
And, we shoud not make magic number in order to make technical debut.
So, I also disagree with this opinion.
```
e.g. the publishing node or that tracking_id/object_id is different from 0 or "".
```

Reply:
@Kukanani OK! This modification is good!
So I will update my comment.

Reply:
@hakuturu583 I'm not too adamant about removing is_tracking. What do you mean by that a a detector, in this instance a tracker, does not do time series processing.

By tracker I understand a program that tracks one object across multiple frames, e.g. a person walking in the image from left to right. The tracker retains state (bounding box in last frame + features) to find the object in the next frame. So the tracker does indeed do time series processing on the series of frames.

Reply:
Maybe I just don't understand the meaning of tracking_id. Assume I track a person walking from left to right in the image in multiple frames: Does each message get the same tracking_id because it is the same person as outputted by the tracker?